plugins/woocommerce/tests/php/src/Internal/DataStores/Orders/OrdersTableQueryTests.php

```diff
@@ -446,4 +446,95 @@ public function test_query_s_argument() {
 		$this->assertCount( 0, $query->orders );
 	}
 
+	/**
+	 * Setup some dummy orders, to help testing the search filter.
```

Comment:
```suggestion
	 * Set up some dummy orders to help test the search filter.
```

(Yes, this is super nitpicky. 😅)

(Not suggesting to change the `setup_` method name also because `set_` as a prefix implies a setter to me.)